### PR TITLE
[processing/pt_br] Adds processing translation to pt_br

### PR DESCRIPTION
**Pull request by kemelzaidan**
- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: pt-br/processing-pt.html.markdown [hunk 1]
```diff
@@ -0,0 +1,479 @@
+---
+language: processing
+filename: learnprocessing.pde
+contributors:
+    - ["Phone Thant Ko", "http://github.com/phonethantko"]
+    - ["Divay Prakash", "https://github.com/divayprakash"]
+translators:
+	- ["Kemel Zaidan", "https://github.com/kemelzaidan"]
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:8]:
```suggestion
    - ["Kemel Zaidan", "https://github.com/kemelzaidan"]
```

File: pt-br/processing-pt.html.markdown [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+
+Processing é uma linguagem de programação para criação de artes digitais e
+conteúdo multimídia, permitindo que não programadores aprendam os fundamentos 
+da programação computacional em um contexto visual.
+
+Embora a linguagem seja baseada na linguagem Java, sua sintaxe foi amplamente
+influenciado por ambas as sintaxes Java e Javascript.
+[Veja mais aqui](https://processing.org/reference/)
+
+A linguagem é tipada estaticamente e também vem com a sua IDE oficial para
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:22]:
```suggestion
A linguagem é tipada estaticamente e também vem com o seu Ambiente de Desenvolvimento Integrado (do inglês Integrated Development Environment - IDE) oficial para
```

File: pt-br/processing-pt.html.markdown [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+   Escrevendo e executando programas em Processing
+   ---------------------------------------
+*/
+
+// No Processing, o ponto de entrada do programa é uma função chamada setup() 
+// com um tipo de retorno void.
+// Observação! A sintaxe é muito semelhante à do C++.
+void setup() {
+  // Isso imprime a saída clássica "Hello World!" no console quando executado.
+  println("Olá Mundo!"); // Outra linguagem cilada para ponto e vírgula, não é?
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:50]: I think this is not a good translation (the original one has a typo also, so it doesn't help). One suggestion (but feel free to fix to some other wording if you want):

```suggestion
  println("Olá Mundo!"); // Mais uma linguagem com esse maldito ponto e vírgula, não é?
```

File: pt-br/processing-pt.html.markdown [hunk 4]
```diff
@@ -0,0 +1,479 @@
...
+    translate(0, -30);
+    rotate(-theta);
+    branch(len);
+    popMatrix();
+
+  }
+}
+```
+
+O processamento é fácil de aprender e é particularmente útil para criar
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:469]:
```suggestion
A linguagem processing é fácil de aprender e é particularmente útil para criar
```

File: pt-br/processing-pt.html.markdown [hunk 5]
```diff
@@ -0,0 +1,479 @@
...
+// Arco
+arc(x, y, largura, altura, inicio, fim, [modo]);
+// Enquanto os primeiros quatro parâmetros são autoexplicativos,
+// início e fim definem os ângulos que o arco começa e termina (em radianos).
+// O parâmetro opcional [mode] define o preenchimento;
+// PIE dá o contorno de torta, CHORD dá o contorno reto e OPEN é como
+// CHORD porém sem contorno
+
+// Curvas
+// O processamento fornece duas implementações de curvas; usando curve() e
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:296]:
```suggestion
// O processing fornece duas implementações de curvas; usando curve() e 
```

File: pt-br/processing-pt.html.markdown [hunk 6]
```diff
@@ -0,0 +1,479 @@
...
+fill(color(0, 0, 0));
+// Se você quiser apenas colorir os contornos das formas, você pode usar
+// função stroke().
+stroke(255, 255, 0, 200); // cor do traço definida para amarelo com transparência
+// definido para um valor menor.
+
+// Imagens
+// O processamento pode renderizar imagens e usá-las de várias maneiras. 
+// Principalmente armazenado como Tipo de dados PImage.
+filter(sombreador); // O processamento suporta várias funções de filtro para manipulação de imagens.
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:383]:
```suggestion
filter(sombreador); // O processing suporta várias funções de filtro para manipulação de imagens.
```

File: pt-br/processing-pt.html.markdown [hunk 7]
```diff
@@ -0,0 +1,479 @@
...
+  }
+}
+```
+
+O processamento é fácil de aprender e é particularmente útil para criar
+conteúdo (mesmo em 3D) sem ter que digitar muitos códigos. É tão simples
+que você pode ler o código e ter uma ideia aproximada do fluxo do programa.
+
+No entanto, isso não se aplica quando você introduz bibliotecas externas, pacotes
+e até mesmo suas próprias aulas. (Confie em mim! O processamento de projetos pode ficar realmente monstruoso...)
```
Comment (mribeirodantas) [inline pt-br/processing-pt.html.markdown:474]:
```suggestion
e até mesmo suas próprias aulas. (Confie em mim! Projetos com processing podem ficar realmente monstruosos...)
```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]
There are many small mistakes, such as translating the name of the language by accident(?). I skimmed over the translation but it'd be great if you can review your translation to make sure no other mistakes passed.

Thanks for the contribution!

File: pt-br/processing-pt.html.markdown [hunk 8]
```diff
@@ -0,0 +1,479 @@
...
+
+// Vetores
+// O Processing fornece uma maneira fácil de implementar vetores em seu ambiente
+// usando a classe PVector. Ela pode descrever um vetor bi ou tridimensional e
+// vem com um conjunto de métodos que são úteis para operações com matrizes.
+// Você pode encontrar mais informações sobre a classe PVector e suas funções 
+// aqui. (https://processing.org/reference/PVector.html)
+
+// Trigonometria
+// O processamento também suporta operações trigonométricas fornecendo um
```
[the remark this answers is not in the document]
Comment (kemelzaidan) [reply pt-br/processing-pt.html.markdown]: Processing should start in capital letter, since it refers to the language name. Fixed in another commit

**Review by kemelzaidan — COMMENTED** [review]
Committing proposed fixes

**Review by mribeirodantas — APPROVED** [review]

**Pull request merged**